core_helpers/logging_objects.py

```diff
@@ -11,6 +11,19 @@ def __init__(self):
         self.msg_list = []
         self.exceptions = []
 
+    @staticmethod
+    def color_text(color, text):
```

Comment:
Can we move the `color` param to the right and have `text` param as first parameter? that way we can add a default value to `color` to `red`.

Reply:
@shivahari Made suggested changes.

Note: Purposely I called color_text method with and without color arg, even-though, I added "red" color as default.